texttospeech/synthesize_file/synthesize_file.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+// Command synthesize_file converts a plain text or SSML file to an audio file.
```

Comment:
Same comments in this file (package comment and func comments).

Reply:
Done